src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/mockservers/CourtRegisterIntegrationTest.kt

```diff
@@ -34,7 +34,7 @@ import java.lang.RuntimeException
 @ActiveProfiles("test")
 @SpringBootTest(
   properties = ["courtRegister.endpoint.url=http://localhost:9095"],
-  classes = [WebClientConfig::class, CourtRegisterClient::class]
+  classes = [TestWebClientConfig::class, CourtRegisterClient::class]
```

Comment:
Why do we need to use `TestWebClientConfig` for this and PrisonRegisterIntegrationTest?

Reply:
Because otherwise when you spin the test up its starts looking for a `MetricsWebClientCustomizer` and `MeterRegistry` to Autowire, and they dont exist.